app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt

```diff
@@ -309,7 +310,7 @@ fun PostActivity(
                     is ApiState.Failure -> ApiErrorText(postRes.msg)
                     is ApiState.Success -> {
                         val postView = postRes.data.post_view
-                        appState.addReturn(PostViewReturn.POST_VIEW, postView.copy(read = true))
+                        if (!account.isAnon()) appState.addReturn(PostViewReturn.POST_VIEW, postView.copy(read = true))
```

Comment:
I think you need the curly braces or klint will balk at it

Reply:
I run ktlint as a precommit and it passed?